packages/cmf/src/interceptors.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable no-console */
+/* eslint-disable no-underscore-dangle */
+const interceptors = [];
```

Comment:
Can we call the files and vars httpInterceptors ? Without any http context, it's not obvious, like in bootstrap() for example